app/scripts/controllers/transactions/index.js

```diff
@@ -543,6 +543,12 @@ export default class TransactionController extends EventEmitter {
       txMeta.txParams.gas = defaultGasLimit;
       txMeta.originalGasEstimate = defaultGasLimit;
     }
+    txMeta.defaultGasEstimates = {
+      gas: txMeta.txParams.gas,
```

Comment:
[Q] Wouldn't this only capture the gasPrice / maxFeePerGas etc of the initial default? If it changes in the gas controller it would be out of sync with what the user is actually being presented, yeah?

Reply:
yes you are right in case of estimate low /medium / high this can get out of sync. Let me change this to use gas estimates